iOS/Extensions/Twilio/TCHChannel+Extensions.swift

```diff
@@ -25,20 +25,18 @@ extension TCHChannel {
     func diffIdentifier() -> NSObjectProtocol {
         return String(optional: self.sid) as NSObjectProtocol
     }
-#warning("Convert to async")
-    func getNonMeMembers() -> Future<[TCHMember], Error> {
-        return Future { promise in
-            if let members = self.members?.membersList() {
-                var nonMeMembers: [TCHMember] = []
-                members.forEach { (member) in
-                    if member.identity != User.current()?.objectId {
-                        nonMeMembers.append(member)
-                    }
+
+    func getNonMeMembers() -> [TCHMember] {
```

Comment:
TCHMember and a User share the same identifier. See if this is used anywhere and remove it. Or add warning todo later. We should only be modifying User objects.